api/graphql/mutations.js

```diff
@@ -26,6 +26,16 @@ function convertGraphqlCreateData (data) {
   }, data))
 }
 
+function convertGraphqlUserSettingsData (data) {
```

Comment:
that makes sense. I'm a little resistant to doing that as I think this code is already awkward and that there is a larger refactor that is close to due here. So it's like: let's not put lipstick on the pig.
But maybe this is how we'll be doing it for awhile, in which case makes sense to standardize the pattern as you suggest.